### PR TITLE
feat: Add operator helm release

### PR DESCRIPTION
**Pull request by adityachoudhari26**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -154,7 +154,8 @@ module "gke_app" {
   local_restore    = var.local_restore
   other_wandb_env = merge({
     "GORILLA_DISABLE_CODE_SAVING"          = var.disable_code_saving,
-    "GORILLA_CUSTOMER_SECRET_STORE_SOURCE" = local.secret_store_source
+    "GORILLA_CUSTOMER_SECRET_STORE_SOURCE" = local.secret_store_source,
+    "GORILLA_GLUE_LIST"                    = var.enable_operator
```
Comment (adityachoudhari26) [inline main.tf:158]: when `GORILLA_GLUE_LIST` is true, it _disables_ glue tasks for that pod. So if operator feature flag is on, disable glue task execution in old pod

File: main.tf [hunk 2]
```diff
@@ -172,3 +173,65 @@ module "gke_app" {
...
+          "GORILLA_DISABLE_CODE_SAVING" = var.disable_code_saving
+          "GORILLA_GLUE_LIST"           = !var.enable_operator
+        }
+      }
+
+      ingress = {
+        issuer = { create = true, provider = "google" }
+        annotations = {
+          "kubernetes.io/ingress.class"                 = "gce"
+          "kubernetes.io/ingress.global-static-ip-name" = module.app_lb.address_operator_name
```
Comment (adityachoudhari26) [inline main.tf:225]: ingress will point to newly provisioned IP address for simplicity

File: main.tf [hunk 3]
```diff
@@ -174,3 +176,81 @@ module "gke_app" {
...
+  source  = "wandb/wandb/helm"
+  version = "1.2.0"
+
+  spec = {
+    values = {
+      global = {
+        host    = local.url
+        license = var.license
+
+        extraEnv = merge({
```
Comment (wandb-zacharyblasczyk) [inline main.tf:190]: Are you missing the inverse GORILLA_GLUE_LIST logic here?

**Review by wandb-zacharyblasczyk — APPROVED** [review]

**Review by wandb-kc — APPROVED** [review]

**Comment by jsbroks** [comment]
This PR is included in version 1.24.0 :tada:

**Pull request merged**